src/utils/api.jsx

```diff
@@ -40,3 +40,10 @@ export function getUsers() {
         return users.data.users
     })
 }
+
+export function deleteComments(comment_id) {
+    return ncNews.delete(`/comments/${comment_id}`)
+    .catch(error => {
+        console.error(error);
+    });
+}
```

Comment:
This should be fine. Please let me know if you have any issues with the changes stated above :)